src/components/help/help.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import Box from '../box';
+import { FontIcon } from '../fontIcon';
+import { Button } from './../toolbox/buttons/button';
+import styles from './help.css';
+import links from './../../constants/help';
+
+/* eslint-disable class-methods-use-this */
+class Help extends React.Component {
+  initOnboarding() {
+    this.props.settingsUpdated({ onBoarding: true });
+  }
+
+  visitHelpCenter() {
+    const win = window.open(links.helpCenter, '_blank');
+    win.focus();
+  }
+
+  render() {
+    const fAQIcon = () => (<span className={styles.inlineIcon}><FontIcon>info</FontIcon><b>{this.props.t(' FAQ')}</b></span>);
+    const chatIcon = () => (<span className={styles.inlineIcon}><FontIcon>conversation</FontIcon><b>{this.props.t(' Chat')}</b></span>);
+    return (
+      <Box className={styles.wrapper}>
+        <aside className={`${grid['col-sm-12']} ${grid['col-md-4']} ${styles.helpSection} ${styles.sideBar}`}>
+          <h2>{this.props.t('Help')}</h2>
+          <p>{this.props.t('Search through our tutorials, FAQs or connect to our community all over the world.')}</p>
+          {
+            this.props.account.address ?
+              <a className='help-onboarding' onClick={() => this.initOnboarding() }>{this.props.t('New to Hub? Take a tour')}
+              <FontIcon>arrow-right</FontIcon>
+            </a> : null
+          }
+        </aside>
+
+        <section className={`${grid['col-sm-12']} ${grid['col-md-8']} ${styles.helpSection} help-articles`}>
+          <article>
+            <h2>
+              <FontIcon className={styles.headerIcon}>logo-icon</FontIcon>
+              {this.props.t('Help Center')}
+            </h2>
+            <div className={styles.articleContainer}>
+              <div>
+                <p>{this.props.t('Search for advice and answers in our extensive ')}
+                  {fAQIcon()}
+                  {this.props.t(' or get in touch in the ')}
+                  {chatIcon()}
```

Comment:
It was missing in the design, but there should be a dot `.` at the end of this sentance.